docs/camino-messenger/bot/installation.mdx

```diff
@@ -32,6 +32,47 @@ software](#compiling-from-source) from its source code, while the second
 method [utilizes Docker](#building-docker-image) to create an image and run a
 container.
 
+## Known Issues
+
+The Camino Messenger Bot the following known issues should be considered:
+
+**Version: 10.2.0**
+
+- After start-up of a fresh bot installation takes 10 to 15 seconds before messages
+  can be sent or received. This period can be more depending on the network 
+  connections and if the bot receives messages before those 15 seconds.
+
+- Empty Message handling: In principle this is a bug in the Partner Plugin, 
+  an empty message should never be attempted to be sent to a partner. The
+  Partner Plugin should sanitize empty messages and we might implement validation
+  for empty messages once we start with the ProtoValidate project.
+
+- Only the first bot returned by the CM-Account will be contacted by other bots.
+  This will be addressed when  High-Availability and Fail-Over will be addressed
+  in the Camino Messenger architecture.
+
+- If the Partner Plugin is disconnected and the bot receives a message, the Bot
+  may crash. To workaround that, use --restart always when running the docker
+  container. 
+
+- Message response header, status field is mandatory and should hold success or
+  failure. This will be enforced when we implement ProtoValidate.
+
+## Networking
+For the bot to connect to other partners properly, allow all outbound ports
+(can connect to internet). Also allow inbound ports which are configured in the
+configuration file.
```

Comment:
Do we really need to open all outbound ports? It should be some specific ports to talk to the Matrix server, no?

Reply:
I'd propose we rephrase this leaving the ports out. Maybe something like:

"For the bot to connect to the camino messenger server, ensure that the network infrastructure and firewall rules are setup properly. The host the bot is running on must be able to resolve DNS names and connect to at least the configured matrix server. For internal communication with the partner plugin make sure the hosts can reach each other and that the firewall rules are allowing the applications to connect to each other."

wdyt?

Reply:
I've updated the page without any port information. But, IMO, we should provide the ports that are needed to be open to reach the Matrix server. I will clarify this and add it. I see it says Matrix clients use 443/8448 on the web, so better verify it from our Matrix config.

Reply:
current example config

supplier bot, not used for distributor

### Partner Plugin
partner_plugin:
  # Partner Plugin hostname and port, should be reachable from this machine.
  # Bot tries to connect to this host and port to relay messages that it receives from
  # the distributors through Matrix Server
  host: localhost:50051

both

### RPC server
rpc_server:
  # Listen on this port for incoming RPC requests
  port: 9090

Reply:
current example config

supplier bot, not used for distributor

partner_plugin:
  # Partner Plugin hostname and port, should be reachable from this machine.
  # Bot tries to connect to this host and port to relay messages that it receives from
  # the distributors through Matrix Server
  host: localhost:50051

both

rpc_server:
  # Listen on this port for incoming RPC requests
  port: 9090